doc/sources/gsg.rst

```diff
@@ -0,0 +1,23 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _get_started:
+
+############################################
+Getting Started Guide
+############################################
+
+Getting Started Guide: https://www.intel.com/content/www/us/en/developer/articles/guide/intel-extension-for-scikit-learn-getting-started.html
```

Comment:
More over we already have Quick Start[](https://huiyan2021.github.io/scikit-learn-intelex/latest/quick-start.html#quick-start) in place. so i don't see point in having links to GSG

Reply:
agree, we can remove GSG. @raoberman , what's your idea?

Reply:
@napetrov , @huiyan2021 we can remove the GSG in a future release - this would require IDZ assistance as we need to redirect the GSG to the documentation. Let's leave it for now to connect the docs and remove it in the future

Reply:
Agree with Rachel, there are some part of the documentation even across other product pages that are in AEM. We need to understand how to consolidate all of the info in github.io as a bigger effort, we can leave the link for now and revisit when we begin the consolidation effort

Reply:
It's fine to keep IDZ page, but for now my point not to make a separate entry here in documentation that just give user a link, while next item is quick starting guide that contain this whole info.  

i.e. - problem is with [doc/sources/gsg.rst](https://github.com/intel/scikit-learn-intelex/pull/1323/files/170b1d510701bf7446b1fb67aa36117a9ae68aa5#diff-bec802dc319c5879ede72e15a85be437954020efa3fe0910c88af75cadb9d66d) this file creation and getting separate menu item for it.  And not with the fact of having IDZ page itself

Reply:
Agree with @napetrov, we do not need this

Reply:
@huiyan2021  i suggest to completely remove [doc/sources/gsg.rst](https://github.com/intel/scikit-learn-intelex/pull/1323/files/170b1d510701bf7446b1fb67aa36117a9ae68aa5#diff-bec802dc319c5879ede72e15a85be437954020efa3fe0910c88af75cadb9d66d)

Not put content there. There is already quick-start available within github docs